examples/ssl/mutual_authentication/main.go

```diff
@@ -24,31 +24,38 @@ import (
 	"github.com/hazelcast/hazelcast-go-client"
 )
 
+// For this code sample, use the provided hazelcast.xml.
+// And make sure server.keystore and server.truststore is accessible by the server,
+
 func main() {
 	// To use SSLConfig with mutual authentication, Hazelcast server should be started with SSL and mutual authentication enabled
 	ctx := context.TODO()
 	var cfg hazelcast.Config
-	cfg = hazelcast.Config{}
-	cfg.Cluster.Network.SetAddresses("foo.bar.com:8888")
+	cfg.Cluster.Network.SetAddresses("localhost:5701")
+	ssl := &cfg.Cluster.Network.SSL
 	// TLS/SSL is enabled
-	cfg.Cluster.Network.SSL.Enabled = true
+	ssl.Enabled = true
+	// we set insecureSkipVerify, since the sample certificates for this example were self-signed.
+	// do not enable it in production.
+	ssl.SetTLSConfig(&tls.Config{InsecureSkipVerify: true})
```

Comment:
Isn't what we do is against "If you need to set a custom tls.Config, for example to turn off verification, you need to assign the tls.Config before any other SSL configuration:"? we are setting ssl config before setting tls config. (ssl.Enabled)

Reply:
Clarified at: https://github.com/hazelcast/hazelcast-go-client/pull/950/commits/ff7272c9f6d6de1cb8f9c78e15dab2027566f282